include/git2/rebase.h

```diff
@@ -30,19 +30,32 @@ typedef struct {
 	unsigned int version;
 
 	/**
-	 * Provide a quiet rebase experience; unused by libgit2 but provided for
-	 * interoperability with other clients.
+	 * Used by `git_rebase_init`, this will instruct other clients working
+	 * on this rebase that you want a quiet rebase experience, which they
+	 * may choose to provide in an application-specific manner.  This has no
+	 * effect upon libgit2 directly, but is provided for interoperability with
+	 * other clients.
```

Comment:
I realise this isn't new in this PR but the "with other clients" irks me. The only time there is a client in the Git system is when you do a fetch or push so using it here would be wrong (and yes I do myself say "client" informally but his is documentation).

Since we're changing the docs here, could we say something like "interoperability between Git tools"?